test/types/Integral/integralvariable.future

```diff
@@ -0,0 +1,3 @@
+error message:  getting internal error when using integral type .
+
+This error message is wrong ,the error should not be internal but rather a user error.
```

Comment:
Being nit-picky, would you mind fixing the space around the comma here, and the period in the first line?